pkg/rbac/testdata/controller.go

```diff
@@ -10,4 +12,9 @@ package controller
 // +kubebuilder:rbac:groups=cron;batch,resources=jobs/status,verbs=get;create
 // +kubebuilder:rbac:groups=batch,resources=jobs/status,verbs=watch;watch
 // +kubebuilder:rbac:groups=art,resources=jobs,verbs=get,namespace=park
-// +kubebuilder:rbac:groups=batch.io,resources=cronjobs,resourceNames=foo;bar;baz,verbs=get;watch
+// +kubebuilder:rbac:groups=batch.io,resources=cronjobs2,resourceNames=foo;bar;baz,verbs=get;watch
```

Comment:
renamed to `cronjobs2` or else it is subsumed by the `kubebuilder:rbac:groups=batch.io,resources=cronjobs,verbs=get;watch;create` line.